pkg/hostagent/ovs.go

```diff
@@ -276,26 +294,6 @@ func (agent *HostAgent) diffPorts(bridges map[string]ovsBridge) []libovsdb.Opera
 			}
 			agent.ignoreOvsPorts[agent.config.IntBridgeName] = []string{"pi-veth_host_ac"}
 			agent.ignoreOvsPorts[agent.config.AccessBridgeName] = []string{"pa-veth_host_ac"}
-			if agent.config.EnableDropLogging {
```

Comment:
Why are we removing this conditional ? Isnt it nice to have it as a config option ?

Reply:
It is not removed. The block that it is moved to has the same condition

Reply:
and the reason it was moved was because overlay condition was added to the existing block.